{% include 'header.html' %}

<link
  href="https://cdn.jsdelivr.net/npm/tabulator-tables@4.9.3/dist/css/tabulator.min.css"
  rel="stylesheet"
/>
<script src="https://cdnjs.cloudflare.com/ajax/libs/PapaParse/5.3.0/papaparse.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/tabulator-tables@4.9.3/dist/js/tabulator.min.js"></script>

<div class="vf-content">
    <h1 class="vf-text vf-text-heading--1">
        About
    </h1>

    <h2 class="vf-text vf-text-heading--2">How the search works</h2>

    <ol class="vf-list vf-list--ordered">
        <li class="vf-list__item">
            When you select a genome, it is processed into a
            <a class="vf-link" href="https://sourmash.readthedocs.io/en/latest/#">sourmash</a>
            signature locally. Your genome never leaves your computer.
        </li>
        <li class="vf-list__item">
          The signature is used to query over 1 million metagenomes in the SRA using
          <a class="vf-link" href="https://github.com/sourmash-bio/branchwater">branchwater</a>
          [<a class="vf-link" href="https://doi.org/10.1101/2022.11.02.514947">paper</a>].
          The quality of the match is represented by containment and containment-based Average Nucleotide Identity (cANI).
        </li>
        <li class="vf-list__item">
          Select SRA metadata is pulled for each of the accessions from a curated
          database and used for plots.
        </li>
    </ol>
</div>

<div class="vf-content">
    <h2 class="vf-text vf-text-heading--2">Search Considerations</h2>

    <p class="vf-text">
        <strong>For best results, search with sequences 50kb or longer.</strong>
        Queries smaller than 50kb may not perform well, and queries smaller than 10kb are likely
        to produce no results at all. These limitations may be addressed in the future; we welcome
        feedback and requests on the branchwater-web
        <a class="vf-link" href="https://github.com/sourmash-bio/branchwater-web/issues">issue tracker</a> on github.
    </p>
</div>

<div class="vf-content">
    <h2 class="vf-text vf-text-heading--2">SRA metadata options</h2>
    <p class="vf-text">
        The list of SRA metadata options and their definitions are a subset of over 900 potential metadata options that are listed in the <a href="https://www.ncbi.nlm.nih.gov/biosample/docs/attributes/">SRA biosample attribute list </a>  and the <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-cloud-based-metadata-table/">SRA cloud-based metadata tables</a> .
        The options presented in the Branchwater web query were selected based on a >4.5% availability cutoff. Some metadata names are similar but represent distinct options, such as the sampling date, because they were derived from information submitted for the SRA run and some are derived from information submitted for the SRA sample (denoted with the '_sam' suffix). Other than minor data reformatting, the metadata provided are directly representative of what is available in the SRA, including missing information or any errors in the SRA submission.
    </p>
</div>

<div class="vf-content">
    <h2 class="vf-text vf-text-heading--2">Who we are</h2>
    <p class="vf-text">
        This instance of Branchwater is run by the MGnify team at EMBL-EBI. <br/>
        MGnify is a bioinformatics resource that offers an automated pipeline for the analysis and archiving of
        metagenomic and metatranscriptomic data. It's part of the European Bioinformatics Institute (EMBL-EBI).
        MGnify provides insights into the diversity and function of microbial communities across a range of
        environments, including oceans, soil, organisms, and the human body. It's a valuable tool for
        researchers studying microbiomes and metagenomes.
    </p>
    <p class="vf-text">
        This web search is powered by sourmash and the result of a collaboration between the Microbiome Informatics team at the European Bioinformatics Institute, The Data Intensive Biology lab at UC Davis, the USDA-ARS Genomics and Bioinformatics Research Unit, and the Joint Genome Institute. <br/>
        Microbiome Informatics Team: Rob Finn, Tatiana Gurbich, Mahfouz Shehu, Alexandar Rogers, Martin Beracochea, Lorna Richardson<br>
        Sourmash Team: Luiz Irber, Titus Brown, Tessa Pierce-Ward
        USDA-ARS: Suzanne Fleishmann, Adam Rivers
        JGI: (hosting support)
    </p>

</div>

    {% include 'footer.html' %}
  </div>
</div>
